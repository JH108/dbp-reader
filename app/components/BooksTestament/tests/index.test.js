import React from 'react';
// import { shallow } from 'enzyme';
import { fromJS } from 'immutable';
import renderer from 'react-test-renderer';
import BooksTestament from '..';

const books = fromJS({});
const handleRef = jest.fn();
const handleBookClick = jest.fn();
const handleChapterClick = jest.fn();
const activeChapter = 1;
const testamentPrefix = '';
const selectedBookName = '';
const activeBookName = '';
const activeTextId = '';
const audioType = '';

describe('<BooksTestament />', () => {
	it('Should match the old snapshot', () => {
		const tree = renderer
			.create(
				<BooksTestament
					books={books}
					handleRef={handleRef}
					handleBookClick={handleBookClick}
					handleChapterClick={handleChapterClick}
					activeChapter={activeChapter}
					testamentPrefix={testamentPrefix}
					selectedBookName={selectedBookName}
					activeBookName={activeBookName}
					activeTextId={activeTextId}
					audioType={audioType}
				/>,
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
