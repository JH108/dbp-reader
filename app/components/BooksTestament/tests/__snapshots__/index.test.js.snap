// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BooksTestament /> Should match the old snapshot 1`] = `null`;
